systemd/openqa-dehydrated.service

```
@@ -0,0 +1,9 @@
+[Unit]
+Description=Renewal openQA certificates.
```

I guess you mean
```suggestion
Description=Renewal of openQA certificates
```
Same in the other file.
Maybe not even "openQA" certificates. I don't know